docs/guides/es2k/compiling-p4-programs.md

```diff
@@ -2,134 +2,107 @@
 
 ## 1. Overview
```

Comment:
It's better to omit the **Overview** heading when the document is part of a user guide. The paragraph(s) before the first heading are understood to be introductory. (This means renumbering the following sections -- or removing the section numbers entirely, which is often appropriate when the document is a section of a user guide.)

Reply:
Removed section numbers.